tests/network/bond/test_l2_bridge_over_bond.py

```diff
@@ -39,16 +39,18 @@ def ovs_linux_br1bond_nad(bridge_device_matrix__class__, namespace):
 def ovs_linux_bond1_worker_1(
     index_number,
     worker_node1,
-    nodes_available_nics,
+    hosts_common_available_ports,
 ):
     """
     Create BOND if setup support BOND
     """
+    if len(hosts_common_available_ports) < 2:
+        pytest.skip("At least two common NICs are required to configure the bond.")
```

Comment:
here and in other places - we should not use "skip" in automation

Reply:
Done.